src/IdentityManager/DataService/Extensions/IdentityServiceCollectionExtension.cs

```diff
@@ -19,20 +21,29 @@ public static class IdentityServiceCollectionExtension
         public static void AddTravelyIdentityService(this IServiceCollection services)
         {
             services.AddScoped<IResourceOwnerPasswordValidator, ResourceOwnerPasswordValidator>()
-                   .AddScoped<IProfileService, ProfileService>()
-                   .AddScoped<IExtensionGrantValidator, DelegationGrantValidator>();
+                   .AddScoped<IProfileService, ProfileService>();
+                   //.AddScoped<IExtensionGrantValidator, DelegationGrantValidator>();
             services.AddTransient<IPasswordHasher<User>, PasswordHasher<User>>();
-        }
-
-        
 
-        public static void InitialConfigIdentityServices(this IServiceCollection services)
-        {
             services.AddIdentityServer()
-                .AddDeveloperSigningCredential()//  AddSigningCredential()
+                .AddDeveloperSigningCredential()
+                                                //.AddSigningCredential()
+                                                .AddPersistedGrantStore<PersistedGrantStore>()
                 .AddInMemoryApiResources(AuthConfigs.GetApiResources())
                 .AddInMemoryClients(AuthConfigs.GetClients())
-                .AddInMemoryApiScopes(AuthConfigs.GetScopes());
+                .AddInMemoryApiScopes(AuthConfigs.GetScopes())
+                ;
+        }
+
+        public static void ConfigureAutoMapper(this IServiceCollection services)
+        {
+            var mapperConfig = new MapperConfiguration(mc =>
```

Comment:
use this package, instead of manual automapper config
https://github.com/AutoMapper/AutoMapper.Extensions.Microsoft.DependencyInjection